apps/api/src/app/shared/utils/auth.utils.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Checks if the authorization header contains a keyless token
+ * @param authorizationHeader - The authorization header value
+ * @returns boolean indicating if the header contains a keyless token
+ */
+export function isKeylessHeader(authorizationHeader: string | undefined): boolean {
+  if (!authorizationHeader) {
+    return false;
+  }
+
+  /*
+   * 'authorization' header 'Keyless pk_keyless_<token>'
+   * 'novu-application-identifier' header 'pk_keyless_<token>'
```

Comment:
Remind me again, can't we just merge all of them to Authorization header?

Reply:
I tried, but I wasn't able to rely solely on the `authorization` header in the end.
We still have a case where a subscriber JWT token is used without any keyless indication.
We need a reliable way to identify that the Inbox client is running in a keyless environment.
Let me know if you have any other ideas or suggestions for how we might solve this.